packages/backend/src/controllers/admin.ts

```diff
@@ -0,0 +1,57 @@
+import { NextFunction, Request, Response, Router } from "express";
+import { Queue } from "../lib/models/queue";
+import { HttpException } from "../lib/errors";
+import { GETQueueReq, GETQueueRes } from "@waiting-line-app/shared-dto/admin";
+import { IQueue, IAdmin } from "@waiting-line-app/shared-dto/db";
+
+const getAdminQueue = async (adminId: string) => {
```

Comment:
move this function to queue.service.ts and change the name to getAllQueuesForAdmin or something clearer